react/features/token-data/TokenData.js

```diff
@@ -0,0 +1,92 @@
+/* @flow */
+
+import jwtDecode from 'jwt-decode';
+
+/**
+ * Represent the data parsed from the JWT token
+ */
+export default class TokenData {
+
+    /**
+     * Object representing callee data.
+     */
+    callee: Object;
+
+    /**
+     * Object representing caller data.
+     */
+    caller: Object;
+
+    /**
+     * Contains group.
+     */
+    group: string;
+
+    /**
+     * Indicates whether user is guest.
+     */
+    isGuest: boolean;
+
+    /**
+     * Raw and encoded JWT token.
+     */
+    jwt: string;
+
+    /**
+     * Payload object from parsed JWT token.
+     */
+    payload: Object;
+
+    /**
+     * Contains server.
+     */
+    server: string;
+
+    /**
+     * Creates instance of token data.
+     *
+     * @param {Object} config - Application config.
+     * @param {string} jwt - The JWT token.
+     */
+    constructor(config: Object, jwt: string) {
```

Comment:
I don't like the passing of config just to check the value. Please move the check out and pass `isGuest` to the constructor.